src/views/LoginView.vue

```diff
@@ -1,35 +1,94 @@
 <template>
   <div>
     <main>
-      <img id="login-image" src="../assets/img/bier.png" alt="logo" />
+      <img
+          id="login-image"
+          src="../assets/img/bier.png"
+          alt="logo"
+      />
       <h1>{{ $t('login.SudoSOS Login') }}</h1>
-      <Button id="login-gewis-button" @click="loginViaGEWIS" severity="success"
-        ><img id="gewis-branding" src="../assets/img/gewis-branding.svg" alt="GEWIS" />{{
-          $t('login.Login via GEWIS')
-        }}</Button
+      <Button
+          id="login-gewis-button"
+          @click="loginViaGEWIS"
+          severity="success"
       >
-      <hr />
-      <form id="login-form" @submit.prevent="ldapLogin">
-        <!--      TODO: Form validation with vee-validate -->
-        <label for="username">{{ $t('login.Username') }}</label>
-        <InputText
-          id="username"
-          type="text"
-          v-bind="username"
-          :placeholder="$t('login.Enter username')"
-        />
-        <label for="password">{{ $t('login.Password') }}</label>
-        <InputText
-          id="password"
-          type="password"
-          v-bind="password"
-          :placeholder="$t('login.Enter password')"
+        <img
+            id="gewis-branding"
+            src="../assets/img/gewis-branding.svg"
```

Comment:
idem

Reply:
Should be fixed too in commit https://github.com/GEWIS/sudosos-frontend-vue3/pull/31/commits/c0cd5afae61160be4c481f76a38752281f930aa6